src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -177,7 +177,7 @@ const Footer = BaseFooter.extend({
 
   handleUpdateFooterLink(data) {
     // only update link for loopback
-    if (!this.isFallbackApproach()) {
+    if (!this.isFallbackApproach() || this.isFallbackDelayed()) {
```

Comment:
is it possible to merge this two functions ?

Reply:
I thought about that, but this is better IMHO. It is hard to categorize the approaches otherwise. Maybe someday, when we have more testing and data, we will add the delay/update footer for all the approaches. By that time, we can rename it better.